test/test-trace-agent.js

```diff
@@ -96,7 +97,7 @@ describe('Trace Agent', function() {
           '123456/667');
         assert(result);
         assert.equal(result.traceId, '123456');
-        assert.equal(result.spanId, '667');
+        assert.equal(result.spanId, 667);
         assert(!result.options);
       });
 
```

Comment:
I've added the tests to the invalid inputs section below. I think it should be a parse error if we get a non-numeric spanId.